src/test/scala/dpla/ingestion3/mappers/providers/NaraMappingTest.scala

```diff
@@ -3,27 +3,27 @@ package dpla.ingestion3.mappers.providers
 
 import java.net.URI
 
+import dpla.ingestion3.mappers.utils.Document
 import dpla.ingestion3.model._
 import dpla.ingestion3.utils.FlatFileIO
 import org.scalatest.{BeforeAndAfter, FlatSpec}
 
-import scala.util.{Failure, Success}
-import scala.xml.XML
+import scala.xml.{NodeSeq, XML}
 
-class NaraExtractorTest extends FlatSpec with BeforeAndAfter {
+class NaraMappingTest extends FlatSpec with BeforeAndAfter {
 
   val shortName = "nara"
   val xmlString: String = new FlatFileIO().readFileAsString("/nara.xml")
-  val xml = XML.loadString(xmlString)
+  val xml: Document[NodeSeq] = Document(XML.loadString(xmlString))
   val itemUri = new URI("http://catalog.archives.gov/id/2132862")
-  val extractor = new NaraExtractor(xmlString, shortName)
+  val extractor = new NaraMapping
 
-  "A NaraExtractor" should "successfully extract from a valid document" in {
-    extractor.build() match {
-      case Success(data) => succeed
-      case Failure(exception) => fail(exception)
-    }
-  }
+//  "A NaraExtractor" should "successfully extract from a valid document" in {
```

Comment:
Probably should just nuke these two commented out tests

Reply:
💥